app/javascript/javascripts/modules/tooltips.js

```diff
@@ -5,4 +5,8 @@
 
 document.addEventListener('DOMContentLoaded', function() {
   $('[data-bs-toggle="tooltip"]').tooltip({ placement: 'right' });
-});
\ No newline at end of file
+});
+
+$(document).on('turbo:load', function() {
```

Comment:
Can we add all events like we do with tom-select?  So the next person trying to add a tooltip to a popover doesn't have todo this all over again :smile: 
https://github.com/hitobito/hitobito/blob/master/app/javascript/javascripts/modules/tom_select.js


https://github.com/hitobito/hitobito/blob/master/app/javascript/controllers/tom_select_controller.js
Maybe you could also write a stimulus controller for this just like tom select. You don't have to switch every tooltip in the whole app to the controller: "tooltip" option, but it may be even cleaner when we implement it to use in the future.

Reply:
Nice! The change from "DOMContentLoaded" to "turbo:load" also doesn't break the `html: true` when loading a page via reload?